modules/aggregation.nf

```diff
@@ -1,19 +1,19 @@
- 
+
 
 process multiqc {
     label 'medium_cpu'
     conda "bioconda::multiqc=1.25"
     publishDir "${params.outputDir}", mode: 'copy'
 
     input:
-        tuple val(email), path('*')
+        path('*')
```

Comment:
+ line 181 in main:
Wouldn't it be potentially handy to have the libraries names available?